bio/strling/merge/meta.yaml

```
@@ -0,0 +1,4 @@
+name: "strling merge"
+description: STRling (pronounced like “sterling”) is a method to detect large STR expansions from short-read sequencing data. "Merge" prepares joint call str loci across all given samples. Requires minimum read evidence from at least one sample. (https://strling.readthedocs.io/en/latest/run.html)
```

As above, I'd suggest:
```suggestion
description: >
    STRling (pronounced “sterling”) is a method to detect large short tandem repeat (STR) expansions from short-read sequencing data.
    ``merge`` prepares joint calling of STR loci across all given samples.
    Requires minimum read evidence from at least one sample.
    Documentation at: https://strling.readthedocs.io/en/latest/run.html
```